release_process.md

```diff
@@ -0,0 +1,52 @@
+# Sinopia Release Process
+
+This is meant to be a guide to DLSS Infrastructure Team on how to release Sinopia to stage and production. This includes the Sinopia Editor and related components (e.g., Sinopia API).
+
+These instructions apply for major releases and dependency updates.
+
+## Checklist
+1. _Skip for a dependency update:_ Product owner prepares release notes.
+  - [ ] Updates `NewsItem.js` component.
+  - [ ] Sinopia [Wiki](https://github.com/LD4P/sinopia/wiki/Latest-Release,-What's-Next)
+2. Create Sinopia Editor release.
+  - [ ] Pull latest main.
+  - [ ] Check out a branch.
+  - [ ] Update the version in *package.json*
+  - [ ] `npm i` to regenerate *package-lock.json* (see note below)
+  - [ ] `npm publish` to publish the version to [npm registry](https://npmjs.com).
+  - [ ] Commit change to branch and push and create PR for version bump.  (You can't push directly to main, since it's a protected branch).
+  - [ ] _Once PR is merged:_ Publish a [new release](https://github.com/LD4P/sinopia_editor/releases/new) with a version like `v1.0.2`.
```

Comment:
While we do tend to wrap GitHub releases around version bumps (e.g., for gems) usually, I wonder if this is a necessary step. Might it be sufficient to create and push a git tag? Or do we prefer GitHub to handle this machinery? It may a six/half-dozen situation.

Reply:
This applies to all codebases.